### DISCUSSION TITLE
How to control canon eos 500D using python

### DISCUSSION THREAD

**Discussion by aswinkalesh** [dukus/digiCamControl#410] [Q&A]
Hello, I need to make a program to control my canon eos 500d for integrating it with pump probe method. I need to interconnect this camera with a controlling stage which carries my high intensity laser. So i just need a program with basic control of the camera suh as shutter speed, aperture and delay coupled with the movement of the stage. Please provide a feedbakc

**Comment by dukus** [comment]
Hi, you can use the https://digicamcontrol.com/doc/userguide/remoteutil
